lib/spack/spack/cmd/list.py

```diff
@@ -23,6 +22,11 @@
 import spack.cmd.common.arguments as arguments
 from spack.version import VersionList
 
+if sys.version_info > (3, 1):
+    from html import escape
+else:
+    from cgi import escape
```

Comment:
According to https://docs.python.org/3.7/library/cgi.html, `cgi.escape` has been deprecated since Python 3.2, and `html.escape` should be used instead. These methods are identical, but `cgi.escape` defaults to `quotes=False`, while `html.escape` defaults to `quotes=True`.